sdk/iotoperations/Azure.ResourceManager.IotOperations/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/iotoperations/Azure.ResourceManager.IotOperations",
+  "Tag": ""
```

Comment:
Please follow https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md to record all tests